docs/timezone.md

```diff
@@ -45,6 +47,51 @@ tz1.getTransitions(now).next().done;  // => true
 tz2.getTransitions(now).next().done;  // => false
 ```
 
+## Temporal.TimeZone.from(thing: string | object) : Temporal.TimeZone
+
+**Parameters:**
+- `thing` (string or object): A `Temporal.TimeZone` object or a string from which to create a `Temporal.TimeZone`.
+
+**Returns:** a new `Temporal.TimeZone` object.
+
+This static method creates a new time zone from another value.
+If the value is a string, it can be:
+- a string that is accepted by `new Temporal.TimeZone()`;
+- a string in the ISO 8601 format including a time zone part, optionally extended with an IANA time zone name in square brackets appended to it;
+- or, a string with _only_ the time zone part of the ISO 8601 format, again extended with optional IANA name in brackets.
```

Comment:
I remain opposed to accepting "-08:00[America/Vancouver]" or "Z". What is the use case? Where would they come from, if not an ISO 8601 string that should be used in entirety?

Reply:
I also feel that these shouldn't be accepted, however, this is the status quo and it seems this is still being debated in #313 (and the precedent is https://github.com/tc39/proposal-temporal/issues/198#issuecomment-547552203); in any case, I've put this on the agenda for the next call.